modules/infra/api/gr_nexthop.h

```diff
@@ -99,6 +100,14 @@ struct gr_nexthop_info_l3 {
 	struct rte_ether_addr mac; //!< link-layer address
 };
 
+#define GR_NH_GROUP_MAX 128
+// Info for GR_NH_T_GROUP nexthops
+struct gr_nexthop_info_group {
+	uint32_t count;
+	uint32_t weights[GR_NH_GROUP_MAX];
+	uint32_t nh_ids[GR_NH_GROUP_MAX];
+};
```

Comment:
How about having an intermediate structure and use a variable size info structure (no need for GR_NH_GROUP_MAX) ?

```c
// Info for GR_NH_T_GROUP nexthops
struct gr_nexthop_group_member {
	uint32_t nh_id;
	uint32_t weight;
};

struct gr_nexthop_info_group {
	uint32_t n_members;
	struct gr_nexthop_group_member members[];
};
```